test/verify/machineslib.py

```diff
@@ -202,12 +202,6 @@ def setUp(self):
         # we don't have configuration to open the firewall for local libvirt machines, so just stop firewalld
         m.execute("systemctl stop firewalld; systemctl try-restart libvirtd")
 
-    def tearDown(self):
```

Comment:
This seems to be the problem -- this overridden method previously failed to call `super.tearDown()`, so it circumvented all the post-tests like unexpected messages and other cleanup. I suggest you put it back for now with a big `FIXME`, and we look at this in a follow-up?